packages/unity-bootstrap-theme/stories/molecules/tabbed-panels/tabbed-panels.js

```diff
@@ -1,94 +1,128 @@
+    const parentNav = parentContainer.querySelector(".nav-tabs");
+
+    // obtenemos el valor del atributo data-scroll-position y nos asguaramos que sea un número entero
+    const scrollPosition = parseInt(parentNav.dataset.scrollPosition, 10);
+    const tabPosition = parentNav.scrollWidth - scrollOffset;
 
+    // ocultamos o mostramos los botones de scroll en función de la posición del scroll
     if (scrollPosition == 0) {
-      parentContainer.find('.scroll-control-prev').hide();
+      parentContainer.querySelector(".scroll-control-prev").style.display =
+        "none";
     } else {
-      parentContainer.find('.scroll-control-prev').show();
+      parentContainer.querySelector(".scroll-control-prev").style.display =
+        "block";
     }
-    if (tabPosition <= parentContainer.width()) {
-      parentContainer.find('.scroll-control-next').hide();
+    if (tabPosition <= parentContainer.offsetWidth) {
+      parentContainer.querySelector(".scroll-control-next").style.display =
+        "none";
     } else {
-      parentContainer.find('.scroll-control-next').show();
+      parentContainer.querySelector(".scroll-control-next").style.display =
+        "block";
     }
   };
 
-  const slideNav = (clicked, e, direction) => {
-    e.preventDefault();
-    const parentNav = $(clicked).siblings('.nav-tabs');
-    let scrollPosition = parentNav.data('scroll-position') * 1;
-    const navItems = parentNav.find('.nav-item').toArray();
-    let scrollOffset = parentNav.css('left').replace('px', '') * 1;
-    var adjustNavItem = 0;
+  // wait to load the page and all resources before initializing
+  window.addEventListener("load", () => {
+    // wait to DOM content is loaded before run these scripts
+    document.addEventListener("click", function (e) {
+      setButtonsCompatibility(e);
+    });
 
-    if (direction == 1 && scrollPosition > 0) {
-      scrollPosition -= 1;
-    }
-    if (scrollPosition < navItems.length - 1 && direction == -1) {
-      scrollPosition += 1;
-    }
-    parentNav.data('scroll-position', scrollPosition);
+    document.querySelectorAll(".uds-tabbed-panels").forEach(item => {
+      const nav = item.querySelector(".nav-tabs");
+      nav.addEventListener("scroll", event => {
+        const scrollPos = event.target.scrollLeft;
+        const prevButton = item.querySelector(".scroll-control-prev");
+        const nextButton = item.querySelector(".scroll-control-next");
+        const atFarRight = nav.offsetWidth + scrollPos + 3 >= nav.scrollWidth;
+        prevButton.style.display = scrollPos === 0 ? "none" : "block";
+        nextButton.style.display = atFarRight ? "none" : "block";
+      });
+    });
 
-    scrollOffset = 0;
-    for (var i = 0; i < scrollPosition; i++) {
-      scrollOffset += $(navItems[i]).outerWidth();
-    }
+    document
+      .querySelector(".scroll-control-next")
+      .addEventListener("click", function (e) {
+        if (window.innerWidth > 992) {
```

Comment:
This might be what is necessary, but I am assuming this is a breakpoint and is likely not to change. Make this a const variable declared of the top of the file